### DISCUSSION TITLE
Image lightbox/preview not opening

### DISCUSSION THREAD

**Discussion by GatorsUF** [lllyasviel/stable-diffusion-webui-forge#1872] [General]
I am having an issue with the expanded image preview not working. Meaning if I click on one of the images in the box on the right side of the txt2img/img2img tabs it won't open the image in a large lightbox/pop-up. Often times it just scrolls to the next image. This problem in very intermittent and sometimes the lightbox opens as expected.

**Comment by zeittresor** [comment]
Hi, i am not sure but might be the same bug i had .. my (not always working) solution was a little modification: https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions/1158
